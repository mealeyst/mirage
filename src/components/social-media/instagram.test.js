import React from 'react'
import { css } from 'styled-components'
import 'jest-styled-components'

import { Instagram } from 'SRC'

const { shallowWithTheme } = global

describe('(Styled Component) Instagram', () => {
  const createInstagram = (props) => {
    return shallowWithTheme(<Instagram {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createInstagram())
    .toMatchSnapshot()
  })
})
