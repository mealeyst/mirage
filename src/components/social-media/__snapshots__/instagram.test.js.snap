// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) Instagram matching the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c0 > a {
  width: 50%;
  max-width: 28rem;
  padding: 1rem;
  box-sizing: border-box;
}

.c0 > a img {
  width: 100%;
  max-width: 26rem;
}

.c0 .gif {
  height: 260px;
  margin-right: -50px;
  z-index: 10;
}

<BaseInstagram
  className="c0"
  limit={4}
/>
`;
